src/psyclone/parse/file_info.py

```diff
@@ -328,13 +393,13 @@ def _cache_save(
 
             # We first remove a potentially existing file
             try:
-                os.remove(self._filepath_cache)
+                os.remove(self._get_filepath_cache())
```

Comment:
Would this file still exist now that it includes the hash in the filename (i.e. if we need to write the cache file, that's because the hash must have changed)? Or are you just being careful?